pkg/controller/pxc/controller.go

```diff
@@ -629,8 +629,10 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 			nodeSet.Spec.Template.Annotations = make(map[string]string)
 		}
 		haProxySet.Spec.Template.Annotations["percona.com/configuration-hash"] = haProxyConfigHash
-		haProxySet.Spec.Template.Annotations["percona.com/ssl-hash"] = sslHash
-		haProxySet.Spec.Template.Annotations["percona.com/ssl-internal-hash"] = sslInternalHash
+		if cr.CompareVersionWith("1.5.0") == 0 {
```

Comment:
should it be `>=` instead of `==`?

Reply:
not, because we add HAproxy into 1.5.0 and we need to add these annotations for HAproxy only for 1.5.0 but not for >= 1.6.0

Reply:
kk)